app/controllers/posts_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
       format.html do
         if @post.save
           flash[:success] = 'Success'
-          redirect_to user_posts_path(current_user, @post)
+          redirect_to user_posts_path(current_admin, @post)
         else
```

Comment:
* When I signup successfully I am not redirected to the root route even after adding some users through the rails console below is a recording of the bug I faced 
![image](https://user-images.githubusercontent.com/52497246/195659062-1e87fe7c-0170-4412-8334-f7938fbcc2fc.png)

Uploading scrnli_10_13_2022_5-54-09 PM.mp4…